pipeline_steps/common.groovy

```diff
@@ -21,6 +21,9 @@ def install_ansible(){
     cp -r /usr/lib64/python2.6/site-packages/selinux .venv/lib64/python2.7/site-packages/ ||:
     source .venv/bin/activate
 
+    # UG-613 change TMPDIR to directory with more space
+    export PREV_TMPDIR=\$TMPDIR
+    export TMPDIR="/var/tmp"
```

Comment:
/var also contains the docker devicemapper datafiles which are now limited to about 4GB. so That leaves ~3 for tmp and anything else thats in /var. Probably more than is there currently, but not by much.